File: Maths/02-FizzBuzz/solutions/askl56-fizzbuzz.rb
def fizzbuzz?(number)
  return 'Fizzbuzz' if number % 15 == 0
  return 'Buzz' if number % 5 == 0
  return 'Fizz' if number % 3 == 0
  number
end

p (1..100).map { |i| fizzbuzz?(i) }

File: Maths/02-FizzBuzz/solutions/askl56-fizzbuzz.rs
fn main() {
    for i in 1..101 {
        match (num % 3 == 0, num % 5 == 0) {
            (true, true)   => println!("FizzBuzz"),
            (true, false)  => println!("Fizz"),
            (false, true)  => println!("Buzz"),
            (false, false) => println!("{}", i),
        }
    }
}
